ansible/roles/i18n/tasks/main.yml

```diff
@@ -11,13 +11,22 @@
 - name: Install ala-i18n apt repository
   apt_repository:
     # We use this repo temporally
-    repo: deb [arch=amd64] https://demo.gbif.es/repo bionic main
+    repo: deb [arch=amd64] https://apt.gbif.es/ bionic main
     state: present
   tags:
     - packages
     - i18n
   when: ansible_os_family == "Debian"
 
+- name: Uninstall demo.gbif.es apt repository
```

Comment:
If the `filename` attribute is specified, it is easier to delete/modify these in the future. We probably don't do this consistently right now, but it is a good practice for the future: 

https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html

Reply:
Done